torch/nn/modules/flatten.py

```diff
@@ -9,6 +9,12 @@
 class Flatten(Module):
     r"""
     Flattens a contiguous range of dims into a tensor. For use with :class:`~nn.Sequential`.
+    
```

Comment:
Since this paragraph is copied from the docs for `torch.flatten`, perhaps we could add a sentence that links readers to the docs for torch.flatten instead :)

Reply:
sensible

Reply:
done!